README.md

```diff
@@ -122,6 +122,14 @@ Then you can run the test suite with,
 If you disable flang support during configure, Fortran tests will be
 disabled and considered as passing the test.
 
+### Cleanning tests
```

Comment:
Cleanning -> Cleaning.

Reply:
Actually. Can you please remove this from the documentation; I think there is no need to document explicitly this feature.

Reply:
Yes, of course.